scan/peers.py

```diff
@@ -194,10 +198,7 @@ def get_state(update: dict, peer_obj: PeerMonitor or None) -> int:
     _data = get_last_cumulative_difficulty()
 
     if update["height"] == _data["height"]:
-        if update["cumulative_difficulty"] == _data["cumulative_difficulty"]:
-            state = PeerMonitor.State.ONLINE
-        else:
-            state = PeerMonitor.State.FORKED
+        state = PeerMonitor.State.ONLINE
```

Comment:
Please do not simply delete, comment out and leave a TODO note.

Reply:
Will do.  Working well in product test fyi.

Reply:
done..